pwa/src/components/handlers/handlerForm.tsx

```diff
@@ -42,6 +42,7 @@ export const HandlerForm: React.FC<HandlerFormProps> = ({ handlerId, endpointId
   const API: APIService = React.useContext(APIContext);
   const [_, setAlert] = React.useContext(AlertContext);
   const [__, setHeader] = React.useContext(HeaderContext);
+  const [documentation, setDocumentation] = React.useState<string>(null);
```

Comment:
You forgot to set the `documentation` in the `Modal`. 

Also, there's a error in this file with the `setAlert()` method that's being called.